WordPress/src/main/java/org/wordpress/android/util/WPLegacyMigrationUtils.java

```diff
@@ -77,4 +93,28 @@ private static String getAccessTokenFromTableThenDelete(Context context, String
         }
         return token;
     }
+
+    private static List<SiteModel> getSelfHostedSitesFromDeprecatedDB(Context context) {
+        List<SiteModel> siteList = new ArrayList<>();
+        try {
+            SQLiteDatabase db = context.getApplicationContext().openOrCreateDatabase(DEPRECATED_DATABASE_NAME, 0, null);
+            String[] fields = new String[]{"username", "password", "url"};
+            String byString = "dotcomFlag=0";
+            Cursor c = db.query(DEPRECATED_BLOGS_TABLE, fields, byString, null, null, null, null);
+            int numRows = c.getCount();
```

Comment:
We should also make sure it's not a jetpack site (we don't want to migrate jetpack sites since they'll now be added by signing in via the wpcom user)